doc/nrf/drivers/hw_cc310.rst

```diff
@@ -1,25 +1,26 @@
 .. _lib_hw_cc310:
+.. _lib_hw_cc3xx:
 
-CC310 hardware driver
+CC3XX hardware driver
```

Comment:
Should the file name be renamed as well?

Reply:
Yes, thank you.